arm-sql/2015-05-01-preview/swagger/virtualNetworkRules.json

```diff
@@ -250,6 +250,7 @@
       "x-ms-azure-resource": true
     },
     "ProxyResource": {
```

Comment:
I am with the service team and we are ok with this for now. Please accept pull request if everything else looks good.